scripts/scripts_object_storage/openstack_swift/local-setup-spark.sh

```diff
@@ -0,0 +1,10 @@
+ #!/usr/bin/env bash
+# Copyright 2016 Yahoo Inc.
+# Licensed under the terms of the Apache 2.0 license.
+# Please see LICENSE file in the project root for terms.
+#
+# This script install Spark locally
+
+wget http://archive.apache.org/dist/spark/spark-2.0.1/spark-2.0.1-bin-hadoop2.7.tgz
```

Comment:
I see your motivation for hadoop-openstack.jar. Placing them into pom.xml will make user's life easier. Otherwise, they will need to download the jar, and add spark-submit argument '-jars'.

Let's introduce a profile specific dependency? We can name the profile as openstack or swift, and that profile will introduce hadoop-openstack as a new dependency.